BankManagerUIApp/BankManagerUIApp/Views/BankView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  BankView.swift
+//  BankManagerUIApp
+//
+//  Created by 김경록 on 11/13/23.
+//
+
+import UIKit
+
+final class BankView: UIView {
+    let addCustomerButton: UIButton = UIButton(title: "고객 10명 추가", titleColor: .systemBlue)
+    let resetButton: UIButton = UIButton(title: "초기화", titleColor: .systemRed)
+    
+    let workTimeLabel: UILabel = UILabel(text: "업무시간 - 00:00.000", fontSize: 23)
+    
+    private let waitingListTitle: UILabel = UILabel(text: "대기중", fontSize: 30, textColor: .white, backgroundColor: .systemGreen)
+    private let workingListTitle: UILabel = UILabel(text: "업무중", fontSize: 30, textColor: .white, backgroundColor: .systemIndigo)
+    
+    let waitingListView: ListView = ListView()
+    let workingListView: ListView = ListView()
```

Comment:
```suggestion
   private let workingListView: ListView = ListView()
```
를 통해서 은닉화를 해주세요. 

은닉화가 되어 있지 않아 외부에서 .text로 변경이 됩니다. 

이 부분은 private으로 은닉화를 하고 updateUI를 함수를 만들어서 진행해주세요.

Reply:
넵 해당 부분 반영했습니다!